conftest.py

```diff
@@ -29,3 +29,8 @@ def _py_files(folder):
 def chdir(tmpdir):
     """Change to pytest-provided temporary directory"""
     tmpdir.chdir()
+
+
+@pytest.fixture()
```

Comment:
We'll also need code that will allow us to skip tests when running with the default reactor, looks like there is no simple way to do that.